src/bundle/Resources/public/js/scripts/helpers/form.error.helper.js

```diff
@@ -1,19 +1,6 @@
 (function (global, doc, ibexa) {
-    const formatLine = (errorMessage) => {
-        const errorIcon = `<svg class="ibexa-icon ibexa-icon--small ibexa-form-error__icon">
-            <use xlink:href="${window.ibexa.helpers.icon.getIconPath('warning-triangle')}"></use>
-        </svg>`;
-        const container = document.createElement('em');
-        const errorMessageNode = document.createTextNode(errorMessage);
-
-        container.classList.add('ibexa-form-error__row');
-        container.insertAdjacentHTML('beforeend', errorIcon);
-        container.append(errorMessageNode);
-
-        return container;
-    };
-
+    // @deprecated, will be removed in 5.0
```

Comment:
I think it would be better to have still formatLine function here and console.log deprecation in case of usage.
Also if we deprecate something we should update all usage of the deprecated function.